README.md

```diff
@@ -32,17 +33,19 @@ adhocracy+ is designed to make online participation easy and accessible to every
     make test
 
 ### Start a local server:
+
     make watch
 
 ### Use postgresql database for testing:
+
 run the following command once:
 ```
-make create-postgres
+make postgres-create
 ```
 to start the testserver with postgresql, run:
 ```
 export DATABASE=postgresql
-make start-postgres
+make postgres-start
```

Comment:
can you add the steps for working with celery? still missing in the README

Reply:
done